myDjangoBlog/settings.py

```diff
@@ -122,3 +123,12 @@
 LOGOUT_REDIRECT_URL = 'home'
 
 SUPERUSER_NAME = 'admin'
+
+AUTHENTICATION_BACKENDS = (
+    'django.contrib.auth.backends.ModelBackend',
+    'blog.authentication.EmailAuthBackend',
+    'social_core.backends.google.GoogleOAuth2',
+)
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = '599560653645-o1oflg2v7iisnkeghvpi6hquibf8m897.apps.googleusercontent.com'
```

Comment:
Sensitive information should not be storing in settings file. Please use env in this case

Reply:
done